src/installation.rst

```diff
@@ -18,7 +18,7 @@ Via Conda (recommended)
 .. admonition:: Supported Python versions
 
    * cylc-flow supports Python 3.8+.
-   * cylc-uiserver currently only supports Python 3.8-3.9.
+   * cylc-uiserver supports Python 3.9+.
```

Comment:
I'd expect the matrix above to include Python 3.10, 11, 12 if this were true?

Reply:
The GH Actions workflow in question is broken - I've disabled it for the time being